packages/accounts/tests/client/accounts_test.coffee

```diff
@@ -0,0 +1,62 @@
+describe("Account header", () ->
+
+  beforeEach((test) ->
+    $('.modal').modal('hide')
+    Meteor.call("clearUsers")
```

Comment:
I'm not sure how it works with a before, but yeah something like that. I don't think you need a setTimeout since you don't need to wait for anything. Once you get to that callback it's done.